Toolset/libraries/revanalyticslibrary.livecodescript

```diff
@@ -0,0 +1,125 @@
+      
+      local tStackName, tScreenName
+      put the short name of tTargetStackID into tStackName
+      put the short name of tTargetCardID into tScreenName
+      revAnalyticsIDEHit "screenview",tStackName,tScreenName
+   end if
+end revAnalyticsScreenChanged
+
+function analyticsSlave
+   return specialfolderpath("support") & slash & "LiveCode" & slash & "LCAnalytics"
+end analyticsSlave
+
+on initialiseAnalytics
+   local tDeployInfo
+   put empty into tDeployInfo
+   
+   put the filename of stack "revAnalytics" into tDeployInfo["stackfile"]
+   put analyticsSlave() into tDeployInfo["output"]
+   
+   local tAuxStacks
+   put the effective filename of stack "revLibUrl" after tAuxStacks
+   
+   repeat for each line tStackFile in tAuxStacks
+      local tTempStackFile
+      put the tempname into tTempStackFile
+      dispatch "revSBResaveScriptOnlyStackAsProperStackFile" to stack "revSBLibrary" with the short name of stack tStackFile, tStackFile, tTempStackFile
+      if tDeployInfo["auxiliary_stackfiles"] is empty then
+         put tTempStackFile into tDeployInfo["auxiliary_stackfiles"]
+      else
+         put return & tTempStackFile after tDeployInfo["auxiliary_stackfiles"]
+      end if
+   end repeat
+
+   put "send" && quote & "revLoadLibrary" & quote && "to stack" && quote & "revLibUrl" & quote into tDeployInfo["startup_script"] 
+   
+   get revEnvironmentRuntimePath()
+   switch the platform
+      case "MacOS"
+         if revEnvironmentIsInstalled() then
+            put it & "/Mac OS X/x86-32/Standalone.app/Contents/MacOS/Standalone" into tDeployInfo["engine"]
+         else
+            if revLicenseType() is "community" then
+               put revEnvironmentBinariesPath() & slash & "Standalone-Community.app/Contents/MacOS/Standalone-Community" into tDeployInfo["engine"]
+            else
+               put revEnvironmentBinariesPath() & slash & "Standalone-Commercial.app/Contents/MacOS/Standalone-Commercial" into tDeployInfo["engine"]
+            end if
+         end if
+         _internal deploy macosx tDeployInfo
+         break
+      case "Windows"
+         if revEnvironmentIsInstalled() then
+            put it & "/Windows/x86-32/Standalone" into tDeployInfo["engine"]
+         else
+            if revLicenseType() is "community" then
+               put revEnvironmentBinariesPath() & slash & "standalone-community.exe" into tDeployInfo["engine"]
+            else
+               put revEnvironmentBinariesPath() & slash & "standalone-commercial.exe" into tDeployInfo["engine"]
+            end if
+         end if
+         _internal deploy windows tDeployInfo
+         break
+      case "Linux"
+         if revEnvironmentIsInstalled() then
+            put it & "/Linux/x86-64/Standalone" into tDeployInfo["engine"]
+         else
+            if revLicenseType() is "community" then
+               put revEnvironmentBinariesPath() & slash & "standalone-community" into tDeployInfo["engine"]
```

Comment:
Since the Business license has been added, all standalone engine files are named in title-case ("Standalone-Community" rather than "standalone-community"). The check should otherwise be okay, though - the business and indy standalone engines still use "commercial" as the suffix.